### PR TITLE
[docs] add an example for transaction execution to graphql docs

### PR DESCRIPTION
**Pull request by emmazzz**
## Description 

As titled

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by amnn — COMMENTED** [review]
This example may be hard to follow for someone who doesn't know how to generate the `txBytes` and `sigs` fields for themselves (this example currently just re-runs a transaction that you have already run).  Could we add some copy at the top of the example explaining how to use the CLI to generate `txBytes` using `--serialize-unsigned-transaction` and then sign it using `sui keytool sign`?

**Review by amnn — COMMENTED** [review]
This example may be hard to follow for someone who doesn't know how to generate the `txBytes` and `sigs` fields for themselves (this example currently just re-runs a transaction that you have already run).  Could we add some copy at the top of the example explaining how to use the CLI to generate `txBytes` using `--serialize-unsigned-transaction` and then sign it using `sui keytool sign`?

File: docs/content/guides/developer/getting-started/graphql-rpc.mdx [hunk 1]
```diff
@@ -333,6 +333,39 @@ fragment DynamicFieldValueSelection on DynamicFieldValue {
   }
 }
 ```
+### Execute a transaction
+
+Transaction execution takes in two arguments, `txBytes` and `signatures`. `txBytes` is the serialized unsigned transaction data, which can be generated using Sui CLI's `--serialize-unsigned-transaction` option. The `signatures` can be generated using Sui CLI's keytool command `sui keytool sign`. More information on Sui CLI can be found [here](/references/cli).
```
Comment (emmazzz) [inline docs/content/guides/developer/getting-started/graphql-rpc.mdx:338]: Is this the right way to reference another doc? @stefan-mysten

Comment (stefan-mysten) [reply docs/content/guides/developer/getting-started/graphql-rpc.mdx:338]: Yes, that looks good!

Comment (stefan-mysten) [reply docs/content/guides/developer/getting-started/graphql-rpc.mdx:338]:
```suggestion
Transaction execution takes in two arguments, `txBytes` and `signatures`. `txBytes` is the serialized unsigned transaction data, which can be generated when using the Sui CLI's `client call` [command](/references/cli/client), to call a Move function by passing the `--serialize-unsigned-transaction` flag. The `signatures` can be generated using Sui CLI's [keytool](/references/cli/keytool) command `sui keytool sign`. More information on Sui CLI can be found [here](/references/cli).
```

**Review by stefan-mysten — APPROVED** [review]
